Source/WebCore/layout/formattingContexts/grid/UnplacedGridItem.h

```diff
@@ -54,11 +55,18 @@ class UnplacedGridItem {
     int explicitRowStart() const;
     int explicitRowEnd() const;
 
+    int normalizedColumnStart() const;
+    int normalizedColumnEnd() const;
+    int normalizedRowStart() const;
+    int normalizedRowEnd() const;
```

Comment:
I'm wondering if these should return unsigned values? Can a normalized value be negative?